### PR TITLE
Use the user provided channel if possible as a default channel

### LINKED ISSUE #5: Can't bookmark pkgs/main channel
Hello, I noticed that I can't bookmark another channel other than conda-forge. For example, opening https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain will instead redirect to https://conda-metadata-app.streamlit.app/?q=conda-forge.

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Fixes #5

This PR fixes the problem where going directly to https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain would redirect to https://conda-metadata-app.streamlit.app/?q=conda-forge.

### CODE REVIEW

**Comment by JeanChristopheMorinPerso** [comment]
@jaimergp do you think you could take a look at this simple PR?

File: app.py [hunk 1]
```diff
@@ -320,6 +320,8 @@ def parse_url_params():
         "Select a channel:",
         CHANNELS,
         key="channel",
+        # Use the user provided channel (via query params) if possible.
+        index=url_params["channel"] in CHANNELS and CHANNELS.index(url_params["channel"]) or 0,
```
Comment (jaimergp) [inline app.py:324]: I prefer this ternary form:

```suggestion
        index=CHANNELS.index(url_params["channel"]) if url_params["channel"] in CHANNELS else 0,
```

**Comment by jaimergp** [comment]
Oops, sorry, didn't see the first time. This is the first community contribution! 🥳

**Comment by JeanChristopheMorinPerso** [comment]
No problem. Thanks for reviewing and merging! I use this service multiple days a day and it's very useful.

**Pull request merged**